rust/aura-core/Cargo.toml

```diff
@@ -27,4 +27,5 @@ serde_json = "1.0"
 srcinfo = "1.1"
 time = { version = "0.3", features = ["serde", "local-offset"] }
 validated = { version = "0.4", features = ["rayon"] }
+versions = { git = "ssh://git@github.com/fosskers/rs-versions.git", rev = "cb2c74823434fe0dc3e7ae09fe1a33bb255c19d3" }
```

Comment:
If you use [url.\<base\>.pushInsteadOf](<https://git-scm.com/docs/git-config#Documentation/git-config.txt-urlltbasegtpushInsteadOf>), you can avoid publishing SSH remote URLs that require authentication.
```gitconfig
[url "ssh://git@github.com/"]
  pushInsteadOf = https://github.com/
```

Reply:
That's just temporary to test a minor change within `versions`.